scripts/mkcloud

```diff
@@ -1077,6 +1077,8 @@ Optional
         It's not deployed by default. Set to 1 to deploy murano.
     want_trove=0 (default=1)
         Deployed by default. Set to 0 to prevent trove from being deployed.
+    want_aodh=0 (default=0)
```

Comment:
I still have doubts about if we should add new option there is already mechanism to skip proposals byt defining `want_name_proposal` https://github.com/SUSE-Cloud/automation/blob/master/scripts/qa_crowbarsetup.sh#L3010.
For me add new option is fine, but some other my PR's were rejected with new `want_` options